src/main/scala/tech/cryptonomic/conseil/tezos/app/TezosNamesOperations.scala

```diff
@@ -81,9 +81,9 @@ class TezosNamesOperations(tnsContracts: TNSContract, node: TezosNodeOperator) e
     blocks =>
       blocks.map { b =>
         val refs = extractAppliedTransactions(b).filter {
-          case Left(op) => tnsContracts.isKnownRegistrar(op.destination)
-          case Right(op) => tnsContracts.isKnownRegistrar(op.destination)
-        }.map(tnsContracts.readLookupMapReference).flattenOption
+          case (Left(op), _) => tnsContracts.isKnownRegistrar(op.destination)
```

Comment:
Here and everywhere else the group hash is not used from code downstream, I would suggest to simply remove it from the list as a first step, to simplify all the code coming after

Reply:
changed